README.md

```diff
@@ -6,6 +6,86 @@
 
 `supportedLanguages.json` is the shared list of all supported locales across FxA
 
+
+### oauth
+
+`oauth.scopes` provides shared logic for validating and checking OAuth scopes.
+
+Detailed documentation on the details of FxA OAuth scope values
+is available from the [fxa-oauth-server](https://github.com/mozilla/fxa-oauth-server/tree/master/docs/scopes.md).
```

Comment:
This link is currently 404, but should be live once the corresponding oauth-server PR lands.  It's currently [here](https://github.com/mozilla/fxa-oauth-server/blob/scopes-documentation/docs/scopes.md#url-scopes).

Is it weird to have the "what is a scope?" documentation in a different place to the "here is the code for working with scopes" documentation?